### PR TITLE
login code

### PR DESCRIPTION
**Pull request by Satishrshinde**

### CODE REVIEW

File: src/login.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    const re = /\S+@\S+\.\S+/;
+    return re.test(email);
+  }
+  function handleSubmit() {
+    if (emailid === "") {
+      setEmailidErr(true);
+    } else {
+      setEmailidErr(false);
+    }
+    if (password === "" || password.length < 8) {
```
Comment (Satishrshinde) [inline src/login.js:17]: removed password.length condition from here.

Comment (visha12l) [inline src/login.js:17]: remove password.length condition from here.

File: src/login.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                invalid password
+              </span>
+            )}
+          </div>
+        </div>
+      </form>
+      <div className="text-center">
+        <button
+          className="btn btn-primary "
+          type="button"
```
Comment (visha12l) [inline src/login.js:75]: remove this line

**Review by visha12l — COMMENTED** [review]
Sign up and reset button should have some spacing

File: src/signup.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+            )}
+            <input type="checkbox" onClick={showPassword} />
+            <span>show password</span>
+          </div>
+        </div>
+      </form>
+      <div>
+        <button
+          onClick={handleSubmit}
+          type="button"
```
Comment (visha12l) [inline src/signup.js:203]: remove this line

**Review by visha12l — APPROVED** [review]

**Pull request merged**